From 4bbca3cf99902cb616fbdbba072c967f901e0129 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Arthur=20Exalta=C3=A7=C3=A3o?=
 <arthur.exaltacao@chainguard.dev>
Date: Tue, 19 Aug 2025 11:03:15 -0300
Subject: [PATCH] freeipa: force platform to fedora
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

FreeIPA doesnt directly support wolfi and that makes it so it errors out
when trying to do anything, this forces it to think that the system is
always fedora

Signed-off-by: Arthur Exaltação <arthur.exaltacao@chainguard.dev>
---
 ipaplatform/osinfo.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ipaplatform/osinfo.py b/ipaplatform/osinfo.py
index 955bc08b7..a245af151 100644
--- a/ipaplatform/osinfo.py
+++ b/ipaplatform/osinfo.py
@@ -94,7 +94,7 @@ class OSInfo(Mapping):
         else:
             raise ValueError("Unsupported platform: {}".format(sys.platform))
         self._info = info
-        self._platform = None
+        self._platform = "fedora"
         self._container = None
 
     def _handle_linux(self):
-- 
2.50.1

